### PR TITLE
Wrong URL to Umbraco forms purchase page

### PR DESCRIPTION
**Pull request by wpplumber**
Ref: 404 => https://umbraco.com/apps/umbraco-forms/

image

### CODE REVIEW

File: umbraco-cms/tutorials/creating-a-basic-website/creating-master-template-part-2.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ We are now going to create a page where we put our contact details. For added fu
 
 Some potential solutions:
 
-* If you're not a programmer you can use the Umbraco built-in package - Umbraco Forms. Umbraco Forms has the added benefit that editors can also create their own forms. Find more information and purchase the product through [Umbraco Apps](https://umbraco.com/apps/umbraco-forms/).
+* If you're not a programmer you can use the Umbraco built-in package - Umbraco Forms. Umbraco Forms has the added benefit that editors can also create their own forms. Find more information and purchase the product through [Umbraco Apps](https://umbraco.com/products/umbraco-forms/).
```
Comment (sofietoft) [inline umbraco-cms/tutorials/creating-a-basic-website/creating-master-template-part-2.md:14]:
```suggestion
* If you're not a programmer you can use the Umbraco built-in package - Umbraco Forms. Umbraco Forms has the added benefit that editors can also create their own forms. Find more information and purchase the product on [Umbraco.com](https://umbraco.com/products/umbraco-forms/).
```

**Review by sofietoft — COMMENTED** [review]
Great catch @wpplumber !
Thanks for correcting this link 💪 

I added a small suggestion, to correct the text with the link as well - that was also a little outdated I noticed!

**Comment by sofietoft** [comment]
Hi @wpplumber ! I took the liberty to make some minor edit to the wording around the link.
I'll get this merged in right away 💪 

Thanks again for helping us fix this link!
I see this is your first-ever PR to the Umbraco Docs - that's amazing! 🎉 
Do you have an account on [Our](https://our.umbraco.com/)? If you do, I can give you a C-Trib badge to show off on your profile.

**Comment by wpplumber** [comment]
> Hi @wpplumber ! I took the liberty to make some minor edit to the wording around the link. I'll get this merged in right away muscle
> 
> Thanks again for helping us fix this link! I see this is your first-ever PR to the Umbraco Docs - that's amazing! tada Do you have an account on [Our](https://our.umbraco.com/)? If you do, I can give you a C-Trib badge to show off on your profile.

Welcome @sofietoft, no problem you have the permission.

Thanks too for all Umbracians which made Umbraco CMS up to this core.

Yes I have an account because I'm an old [member](https://our.umbraco.com/members/wpplumber/) and thank you for offering this badge.

**Comment by sofietoft** [comment]
That's amazing to hear 😄 
I've granted you the C-trib bagde - You're now part of the contributors to the project as well 👏 🥳 

Hope to see you back here with more great stuff!
Cheers!

**Comment by wpplumber** [comment]
Received.

image

> Hope to see you back here with more great stuff!
> Cheers!

Yes, in-chae-allah.

Thank you Sofie Toft

**Pull request merged**